prow/config.yaml

```diff
@@ -545,3 +545,16 @@ presets:
   volumeMounts:
   - name: docker-graph
     mountPath: /docker-graph
+# experimental preset for dind, prefer the one above
+- labels:
+    preset-dind-memory-backed: "true"
+  env:
+  - name: DOCKER_IN_DOCKER_ENABLED
+    value: "true"
+  volumes:
+  - name: docker-graph
+    emptyDir:
+      medium: "Memory"
```

Comment:
this is otherwise same as the preset above, but the docker graph emptyDir is memory backed instead